sources/updater/signpost/src/set.rs

```diff
@@ -30,8 +30,8 @@ impl fmt::Display for PartitionSet {
     }
 }
 
-#[derive(Debug, Clone, Copy)]
-pub(crate) enum SetSelect {
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum SetSelect {
     A,
     B,
 }
```

Comment:
Do we still need to widen access in signpost's `set.rs` and `state.rs`?

Reply:
I still need it to be public for determining which partition is next to boot.